attributes/default.rb

```diff
@@ -356,6 +356,7 @@
   default['kkafka']['broker']['super']['users']                                       = "User:hopsworks0;User:#{node['kkafka']['user']}"
 end
 
+default['kkafka']['broker']['consumer_offsets']['enabled']                            = "false"
```

Comment:
Sorry for the extra work, but the name of this setting is a bit confusing, can we call it something like
```suggestion
default['kkafka']['broker']['consumer_offsets']['access_allowed']                            = "false"
```